patterns/roks/variables.tf

```diff
@@ -281,6 +282,30 @@ variable "operating_system" {
   }
 }
 
+variable "use_private_endpoint" {
+  type        = bool
+  description = "Set this to true to force all cluster related api calls to use the IBM Cloud private endpoints."
+  default     = false
+}
+
+variable "verify_worker_network_readiness" {
+  type        = bool
+  description = "By setting this to true, a script will run kubectl commands to verify that all worker nodes can communicate successfully with the master. If the runtime does not have access to the kube cluster to run kubectl commands, this should be set to false."
+  default     = false
+}
+
+variable "ignore_worker_pool_size_changes" {
+  type        = bool
+  description = "Enable if using worker autoscaling. Stops Terraform managing worker count"
+  default     = false
+}
+
+variable "cluster_config_endpoint_type" {
```

Comment:
Because we only have the options of a private-only cluster at the pattern level, this input is not really needed becasue setting to something like "vpe" might fail due to ACL issues? I think you can just default this in the config.tf to "default" or "private", I'm not sure giving this option to the user is needed in the pattern.